src/sql/workbench/browser/modal/optionsDialog.ts

```diff
@@ -178,6 +178,10 @@ export class OptionsDialog extends Modal {
 		}
 	}
 
+	public get options(): IOptionsDialogOptions {
```

Comment:
does this need to be public?

Reply:
We don't have to expose this necessarily. Let me make this private. We can expose it if needed later.